partiql-tests-runner/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -0,0 +1,5 @@
+distributionBase=GRADLE_USER_HOME
```

Comment:
The following should be deleted
- `gradle/wrapper` 
- gradlew
- gadlew.bat

Because we are included partiql-tests-runner as a module of the partiql-lang-kotlin project, it does not need its own wrapper